src/Voron/Slice.cs

```diff
@@ -302,6 +302,8 @@ public override int GetHashCode()
 
         public override string ToString()
         {
+            if (this.Content[^1] == '\0')
+                return this.Content.Substring(this.Size - 1);
```

Comment:
This should push to Utf8 encoding as the default version.

Reply:
Yeah, why is that here at all?

Reply:
I was working with Slices and found out it's always adding null character (so all comparisons failed in my case). Removed because we don't need it anymore.